qmi/core/transport.py

```diff
@@ -262,7 +280,7 @@ def _parse_positional_parameters(self, params: List[str]) -> Mapping[str, Any]:
                                                        ty, param)
         return d
 
-    def _parse_keyword_parameters(self, strings: List[str]) -> Mapping[str, Any]:
+    def _parse_keyword_parameters(self, strings: List[str]) -> Dict[str, Any]:
```

Comment:
in this method there's a type check that I cannot comment on. Its line 294:
```python
if ty == int ...
```
this can be changed to `if ty is int`
Only if you want to boyscout a bit more

Reply:
OK, changing it, why not :)